spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -303,8 +308,14 @@ def step(  # noqa
             if np.count_nonzero(base_action) > 0:
                 # Command velocities using the input action
                 lin_dist, ang_dist = base_action
-                lin_dist *= self.max_lin_dist
-                ang_dist *= self.max_ang_dist
+                if max_lin_dist_mobile_gaze is not None:
+                    lin_dist *= self.config[max_lin_dist_mobile_gaze]
+                else:
+                    lin_dist *= self.max_lin_dist
+                if max_ang_dist_mobile_gaze is not None:
+                    ang_dist *= self.config[max_ang_dist_mobile_gaze]
+                else:
+                    ang_dist *= self.max_ang_dist
```

Comment:
This can be a class member variable defined inside init

Reply:
@jimmytyyang

Reply:
okay, fixed this! thank you!